examples/DotNetCore/ConsoleAppWithTableStorageReference/Program.cs

```diff
@@ -0,0 +1,127 @@
+﻿using Azure.Data.AppConfiguration;
+using Azure.Data.Tables;
+using Azure.Identity;
+using System.Text;
+using System.Text.Json;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Examples.ConsoleAppWithTableStorageReference
+{
+    class Program
+    {
+        static IConfiguration Configuration { get; set; }
+
+        static void Main(string[] args)
+        {
+            Configure();
+
+            StringBuilder sb = BuildProductTable();
```

Comment:
```suggestion
            var products = new List<Product>();

            Configuration.GetSection("MyShop:Inventory").Bind(products)
            
            Console.Clear();

            Console.WriteLine(FormatProducts(products));
            
            Console.WriteLine("\nPress any key to exit...");

            // Finish on key press
            Console.ReadKey();
```

Reply:
Can `Configuration.GetSection("MyShop:Inventory").Bind(products)` become `Configuration.GetSection("MyShop:Inventory").Get<List<Product>>()`? I seem to remember such an API.